pvlib/irradiance.py

```diff
@@ -1797,44 +1785,45 @@ def disc(ghi, solar_zenith, datetime_or_doy, pressure=101325,
     Parameters
     ----------
     ghi : numeric
-        Global horizontal irradiance. [Wm⁻²]
+        Global horizontal irradiance. See :term:`ghi`. [Wm⁻²]
 
     solar_zenith : numeric
-        True (not refraction-corrected) solar zenith angles in decimal
-        degrees.
+        True (not refraction-corrected) solar zenith angles. See
+        :term:`solar_zenith`. [°]
 
     datetime_or_doy : int, float, array, pd.DatetimeIndex
         Day of year or array of days of year e.g.
         pd.DatetimeIndex.dayofyear, or pd.DatetimeIndex.
 
     pressure : numeric or None, default 101325
-        Site pressure in Pascal. Uses absolute (pressure-corrected) airmass
-        by default. Set to ``None`` to use relative airmass.
+        Site pressure. Uses absolute (pressure-corrected) airmass
```

Comment:
This looks like an old copy/paste problem. Isn't pressure just an input? I don't think that it is calculated from air mass.